components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -276,4 +312,42 @@ class GeckoEngine(
             this.fontSizeFactor = it.fontSizeFactor
         }
     }
+
+    private fun ContentBlockingController.LogEntry.BlockingData.hasBlockedCookies(): Boolean {
```

Comment:
Since these are extensions, they probably don't need to live inside the GeckoEngine class. At the root of the file should also work.

Reply:
👍

Reply:
👍